client/layout/masterbar/logged-in.jsx

```diff
@@ -186,7 +206,7 @@ class MasterbarLoggedIn extends React.Component {
 				<Item
 					tipTarget="reader"
 					className="masterbar__reader"
-					url="/read"
+					url={ config.isEnabled( 'nav-unification' ) ? '/read?flags=nav-unification' : '/read' }
```

Comment:
The flag doesn't persist on relative URLs?

Reply:
No, it doesn't. Why should it? I am not sure I am aware of doing this without JS